spec/client/services/columnAssignmentService.spec.js

```diff
@@ -0,0 +1,80 @@
+require("phantomjs-polyfill");
+
+describe("columnAssignmentService", function() {
+
+    var columnAssignmentService;
+
+    beforeEach(function() {
+        angular.module("ngMaterial", []);
+        angular.module("angularMoment", []);
+        angular.module("ngSanitize", []);
+        module("TwitterWallApp");
+    });
+
+    beforeEach(inject(function(_columnAssignmentService_) {
+        columnAssignmentService = _columnAssignmentService_;
+    }));
+
+    var testColumnDataList;
+    var testTweets;
+
+    beforeEach(function() {
+        var x = new(columnAssignmentService.ColumnData)(2, function(tweet) {
+            return tweet.pinned === true;
+        }, function(tweetA, tweetB) {
+            return tweetA.pinTime - tweetB.pinTime;
+        });
+        testColumnDataList = [
+            new columnAssignmentService.ColumnData(2, function(tweet) {
+                return tweet.pinned === true;
+            }, function(tweetA, tweetB) {
+                return tweetA.pinTime - tweetB.pinTime;
```

Comment:
these sort functions are testing 'pinTime' but the testTweets are defining 'pinnedTime'